pkg/kubectl/cmd/convert/BUILD

```diff
@@ -2,12 +2,30 @@ load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
 
 go_library(
     name = "go_default_library",
-    srcs = ["convert.go"],
+    srcs = [
+        "convert.go",
+        "import_known_versions.go",
+    ],
     importpath = "k8s.io/kubernetes/pkg/kubectl/cmd/convert",
     visibility = ["//visibility:public"],
     deps = [
         "//pkg/api/legacyscheme:go_default_library",
+        "//pkg/apis/apps/install:go_default_library",
```

Comment:
Agree with @soltysh. Not sure why we're adding these dependencies on internal types here. This is problematic. BTW, we are trying to deprecate `convert` or move it into a plugin.

Reply:
> move it into a plugin

this one sounds better to me